taxi/views.py

```diff
@@ -87,3 +89,37 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+    template_name = "taxi/driver_create.html"
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+    template_name = "taxi/driver_update_license.html"
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+    template_name = "taxi/driver_delete.html"
+
+
+def add_user(request, car_id: int) -> HttpResponse:
+    car = Car.objects.get(id=car_id)
+    if request.user not in car.drivers.all():
+        car.drivers.add(request.user)
+    return redirect("taxi:car-detail", pk=car.id)
+
+
+def delete_user(request, car_id: int) -> HttpResponse:
+    car = Car.objects.get(id=car_id)
```

Comment:
Consider handling the case where the Car object with the given `car_id` does not exist. You can use `get_object_or_404` instead of `Car.objects.get` to automatically return a 404 response if the object is not found.